easybuild/easyconfigs/a/Arrow/Arrow-0.17.1-intel-2020b.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '0.17.1'
+
+homepage = 'https://arrow.apache.org'
+description = """Apache Arrow (incl. PyArrow Python bindings)), a cross-language development platform
```

Comment:
```suggestion
description = """Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
```

Reply:
@boegel I have to fix this typo, or I won't sleep at night

Reply:
:+1: